<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogCut</class>
 <widget class="QDialog" name="CDialogCut">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>剪切</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: rgb(100, 100, 100);}
QLabel{color:rgb(255, 255, 255)}

QWidget
{
	font-size:15px;
	font-family:Microsoft Yahei;	
}
</string>
  </property>
  <widget class="QFrame" name="frameSysBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-image:url(:/Skin/bg-above2.png);
}
QLabel
{
border-image:null;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="logo">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Skin/logo.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTitle">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: rgb(255, 255, 255);
	font-size:18px;
	font-family:Microsoft Yahei;	
}</string>
        </property>
        <property name="text">
         <string>剪切</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_sysmenu">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{border-image:null;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="BtnClose">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>关闭</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/close.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/close.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/close.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/close.png)90 0 0 0 repeat
}
</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BtnMin">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>最小化</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/min.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/min.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/min.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/min.png)90 0 0 0 repeat
}
</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelFileName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnStartCut">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>638</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
   </property>
   <property name="text">
    <string>开始剪切</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCutStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>837</x>
     <y>638</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>643</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
border:2px solid grey;
border-radius:5px;
text-align: center;
}
QProgressBar:disabled{
	background-color: rgb(158, 158, 158);
}

QProgressBar::chunk {
background-color:#05B8CC;
width:20px;

}
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QFrame" name="frameTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>开始时间:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结束时间:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeEditStart">
       <property name="displayFormat">
        <string>H:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QTimeEdit" name="timeEditStop">
       <property name="displayFormat">
        <string>H:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="BtnTimeStart">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-stop.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-stop.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-stop.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-stop.png)90 0 0 0 repeat
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="BtnTimeStop">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-stop.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-stop.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-stop.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-stop.png)90 0 0 0 repeat
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frameComboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BtnFileOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
		font-size:17px;
	font-family:Microsoft Yahei;
	color:white;
    border-width: 0px;
}

QPushButton
{
	
	border-image: url(:/Skin/button.png)0 0 repeat;

}
QPushButton:hover
{	
	border-image: url(:/Skin/button.png)31 0 0 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/button.png)62 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/button.png)93 0 0 0 repeat
}</string>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameBackGround">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Skin/cut.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_5"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>931</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{border:1px solid rgb(120,120,120);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="framePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>保存路径:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditDstPath"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="BtnFilePath">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelState">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>646</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>639</y>
     <width>64</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>剪切进度:</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frameSysBar</zorder>
  <zorder>BtnStartCut</zorder>
  <zorder>BtnCutStop</zorder>
  <zorder>progressBar</zorder>
  <zorder>frameTime</zorder>
  <zorder>frameComboBox</zorder>
  <zorder>framePath</zorder>
  <zorder>labelState</zorder>
  <zorder>label_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sisecplayer.qrc"/>
 </resources>
 <connections/>
</ui>
